content/docs/hello-world.md

```diff
@@ -15,36 +15,43 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Het weergeeft een kop met de tekst "Hello, world!" op de pagina.
 
-[](codepen://hello-world)
+[Probeer het op CodePen](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Klik op de bovenstaande link om een online editor te openen.
+Voel je vrij om wat aanpassingen te maken en te zien hoe deze de uitvoer beïnvloeden.
+De meeste pagina's in deze gids hebben dit soort bewerkbare voorbeelden.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Hoe Lees Je Deze Gids {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+In deze gids zullen we de bouwstenen van React-applicaties onderzoeken: elementen en componenten.
+Van zodra je ze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
 
 >Tip
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Deze gids is bedoeld voor zij die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Mogelijk vind je dan dat deze gids en de tutorial elkaar aanvullen.
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+Dit is het eerste hoofdstuk in een stap-voor-stapgids over de belangrijkste React-concepten.
+Je kan een lijst van alle hoofdstukken vinden in de navigatiezijbalk.
+Als je dit leest vanaf een mobiel apparaat, kan je de navigatie openen door op de knop in de rechterbenedenhoek van je scherm te drukken.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Elk hoofdstuk in deze gids bouwt verder op de kennis die in voorgaande hoofdstukken werd opgebouwd.
+**Het grootste deel van React kan je leren door de hoofdstukken “Algemene Concepten” te lezen in de volgorde waarin ze verschijnen in de zijbalk**.
```

Comment:
```suggestion
**Het grootste deel van React kan je leren door de “Algemene Concepten” hoofdstukken te lezen in de volgorde waarin ze verschijnen in de zijbalk**.
```

Reply:
I think there'd technically be a hyphen due again with this change. Waiting for the result of that discussion.